linkcheck/configuration/__init__.py

```diff
@@ -428,43 +428,45 @@ def get_user_config():
     return userconf
 
 
-def get_gconf_http_proxy():
-    """Return host:port for GConf HTTP proxy if found, else None."""
+def get_dconf_http_proxy():
+    """Return host:port for dconf HTTP proxy if found, else None."""
     try:
-        import gconf
+        import gi
+        from gi.repository import Gio
```

Comment:
I often see warnings telling me to call `gi.require_version('Gio', '2.0')` and such before importing stuff from gi.repository.

Reply:
Gio is the one I don't see those warnings for, but maybe it is safer. I'll take it for the reasons below anyway.